src/D2686.tex

```diff
@@ -314,7 +315,7 @@ \section{Next step}
 
 \section{Wording for constexpr structured binding}
 
-\ednote{This wording only makes constexpr structured a valid grammatical construct and does not permit automatic storage duration bindings of tuple-like objets}
+\ednote{This wording makes constexpr structured only as a valid grammatical construct and does not permit automatic storage duration bindings of tuple-like objects.} % lah: Something isn't quite right in this sentence. I've changed it to what I think you intended, but reread it carefully. 
```

Comment:
I should clarify that this is option 0 wording.
ie it allows constexpr structured binding but it will not work for the tuple case. so your suggestion is not quite right either but I'll try to clarify.